schemas/data/record.schema.json

```diff
@@ -21,6 +21,14 @@
           "type": "string",
           "format": "uri-reference",
           "description": "A unique identifier for the record."
+        },
+        "xdm:eventList": {
+          "title": "Event List",
+          "description": "List of time-series based events that relate to schemas based on record.",
+          "type": "array",
```

Comment:
Given that this a field for time-series events, what is the expected order desc/asc?

Reply:
@tcuevas-adobe I don't have any particular opinion on the ordering, so please outline what you consider the better option.

I can see the following alternatives:
1. The schema defines one (ascending/descending).
2. There is a descriptor that defines this.
3. The query system takes this as a parameter when it constructs the profile record and the schema may have a property for `eventListOrder` to represent this.

I think the latter is the better option.